test/unit/core/store/middleware.ts

```diff
@@ -175,10 +182,49 @@ suite('Middleware', ({ expect, spy, stub }) => {
     });
   });
 
+  describe('checkPastPurchaseSkus()', () => {
+    it('should pass action through if not in PAST_PURCHASE_SKU_ACTIONS', () => {
+      const action = { type: 'NOT_ACTUALLY_AN_ACTION' };
+
+      Middleware.checkPastPurchaseSkus(<any>{})(null)(next)(action);
+
+      expect(next).to.be.calledWith(action);
+    });
+
+    it('should pass action through if past purchases present', () => {
+      const action = { type: PAST_PURCHASE_SKU_ACTIONS[0] };
+      const state = 'st';
```

Comment:
please retain base value types, this can just be an object with keys like `{ a: 'b' }`
The reason I try to give them different keys and values for every variable in a function is because it makes it easier to debug when you can see unique keys for each variable. I don't expect you to go through and change all of your variables that are using the same values within a function, but just so that going forward you understand that pattern better